airsonic-main/src/main/webapp/WEB-INF/jsp/podcastChannel.jsp

```diff
@@ -70,28 +179,46 @@
             return result;
         }
 
+        function getSelectedEpisodesMediaId() {
+            var result = [];
+            for (var i = 0; i < ${fn:length(model.episodes)}; i++) {
+                var checkbox = $("#episode" + i);
+                if (checkbox.is(":checked")) {
+                    if (podcastEpisodes[i].status == 'COMPLETED') {
+                        result.push(podcastEpisodes[i].mediaFileId);
+                    } else {
+                        $().toastmessage("showErrorToast", "Download of the episode needs to be completed for that action");
+                    }
+                }
+            }
+            return result;
+        }
+
     </script>
 </head>
 <body class="mainframe bgcolor1" onload="init()">
 
 <div style="float:left;margin-right:1.5em;margin-bottom:1.5em">
-<c:import url="coverArt.jsp">
-    <c:param name="podcastChannelId" value="${model.channel.id}"/>
-    <c:param name="coverArtSize" value="200"/>
-</c:import>
+    <c:import url="coverArt.jsp">
+        <c:param name="podcastChannelId" value="${model.channel.id}"/>
+        <c:param name="coverArtSize" value="200"/>
+    </c:import>
 </div>
 
-<h1 id="name"><a href="podcastChannels.view"><fmt:message key="podcastreceiver.title"/></a> &raquo; ${fn:escapeXml(model.channel.title)}</h1>
+<h1 id="name"><a href="podcastChannels.view"><fmt:message key="podcastreceiver.title"/></a>
+    &raquo; ${fn:escapeXml(model.channel.title)}</h1>
 <h2>
-    <span class="header"><a href="javascript:top.playQueue.onPlayPodcastChannel(${model.channel.id})"><fmt:message key="common.play"/></a></span>
+    <span class="header"><a href="javascript:top.playQueue.onPlayPodcastChannel(${model.channel.id})"><fmt:message key="main.playall"/></a></span>
```

Comment:
Updated this to show "Play all" instead of "Play", which is clearer and more consistent